models/demos/ttnn_falcon7b/tt/falcon_attention.py

```diff
@@ -165,7 +162,9 @@ def __call__(
                 attn_weights = ttnn.experimental.operations.primary.transformers.attn_matmul(
                     query_layer,
                     key_layer_transposed,
-                    compute_with_storage_grid_size=ttnn.experimental.tensor.CoreCoord(8, 7),
+                    compute_with_storage_grid_size=ttnn.experimental.tensor.CoreCoord(
+                        self.core_grid.x, self.core_grid.y
+                    ),
```

Comment:
We should consolidate `ttnn.CoreGrid` and `ttlib.CoreCoord` eventually. There's really not much difference, except that `ttnn` explicitly only uses coordinates as a grid so it's name differently.

Reply:
yeah, agreed. Tracking here: https://github.com/tenstorrent/tt-metal/issues/9124